app/views/payments/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for([@category, @payment]) do |f| %>
+  <div class='form-head'>
+    <%= link_to categories_path do %>
+      <i class="fa fa-arrow-left"></i>
+    <% end %>
+    <h2>Payments</h2>
+    <div class="actions">
+      <%= f.submit "Next", class: 'btn', style: 'color:white; background-color:transparent;' %>
+    </div>
+  </div>
+
+  <div class="form-floating">
+    <%= f.text_field :name, class: 'form-control', id: 'floatingInput', placeholder: 'education' %>
+    <%= f.label :name, "Name", class: 'form-label', for: 'floatingInput' %>
+  </div>
+
+  <div class="form-floating">
+    <%= f.text_field :amount, class: 'form-control', id: 'floatingInput', placeholder: 'nike.png' %>
+    <%= f.label :amount, "Amount", class: 'form-label', for: 'floatingInput' %>
+  </div>
+
+  <%= f.hidden_field :user_id, value: @user.id %>
+  <%= f.hidden_field :category_id, value: @category.id %>
```

Comment:
- [ OPTIONAL] Amazing job working on the requirements so far, I think you made huge efforts on the project but you still have one small thing to proceed, so here when you add a new transaction I think you added the name and the amount inputs but it is mentioned to add an input for a category so you let the user be able to choose the category that he spent this specific transaction on it. so I think it would be great if you add it 🤩✨🚀💯✅ but I assign it as optional because the previous reviewer didn't mention it but it still important to make it ✨


![image](https://github.com/Simpleshaikh1/BudgetBuddy/assets/58553711/ffdea763-9c47-4478-9bf0-c80969f98820)